src/index.js

```diff
@@ -1,8 +1,15 @@
-var content = require('./populate-content');
 var widget = require('./widget');
 
-content.getContent(['models', 'examples']);
 
 $(document).ready(function (){
+    var search = require('./components/search');
+    var browse = require('./components/browse');
+    var model = require('./components/models');
+    var example = require('./components/examples');
+
     widget.init();
-})
+    var cardWidget = require('./card-widget');
```

Comment:
totally a coding style thing, can we group all the requires together and do the widget.init() at the very end?

Reply:
Yeah, I did it this way because I was under the impression that order mattered when loading in these files (as it did with the plain js version), but it seems this is not the case here as I tried locally and it worked!